deps/chakrashim/lib/chakra_shim.js

```diff
@@ -89,12 +120,34 @@
   // Parse 'stack' string into StackTrace frames. Skip top 'skipDepth' frames,
   // and optionally skip top to 'startName' function frames.
   function parseStack(stack, skipDepth, startName) {
+    var curr = parseStack;
+
     var splittedStack = stack.split('\n');
-    splittedStack.splice(0, skipDepth + 1); // also skip top name/message line
     var errstack = [];
 
     for (var i = 0; i < splittedStack.length; i++) {
       var parens = /\(/.exec(splittedStack[i]);
```

Comment:
We could, but we need parens to get the index down, so just reused `parens` instead.